modules/nf-core/quantmsutils/mzmlstatistics/main.nf

```diff
@@ -0,0 +1,59 @@
+process QUANTMSUTILS_MZMLSTATISTICS {
```

Comment:
As the name suggests, this module only works for mzML files, no `.d` or `.raw` files. We might want to address this in the subworkflow